codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/HttpProtocolTestGenerator.java

```diff
@@ -332,7 +332,8 @@ private void renderResponseMiddleware(HttpResponseTestCase testCase) {
 
     private void renderResponseMiddlewareBody(Optional<String> body) {
         if (body.isPresent()) {
-            writer.write("response.body = StringIO.new('$L')", body.get());
+            writer.write("response.body.write('$L')", body.get());
```

Comment:
I prefer the `<<` method instead of write. It's cleaner and we can drop a set of parens. Thoughts?

Reply:
I prefer the `write` method.  Its the counter to `read` and in a lot of places we define our own IO like objects -  I don't want to have to remeber to define both a write and a << method.  Ruby obviously doesn't define an actual IO interface that we can count on though...